### DISCUSSION TITLE
Accessibility - Dragon Tool - Focus not going to search box on voice over

### DISCUSSION THREAD

**Discussion by MariaAug** [microsoft-search/pnp-modern-search#3035] [Ideas]
**Version used**
4.5.3

**Describe the bug**
Dragon tool does not recognise the PnP search box if voice command is provided as 'Click <placeholder text>'. Eg: Click Search News

**To Reproduce**

1. Add PnP search box web part on a SharePoint page.
2. Edit web  part. Provide a placeholder text for the search box. Example: Search news
3. Save the page.
4. Provide the voice command using Dragon tool, 'Click search news'.
5. The focus does not go to the PnP search box 

**Expected behavior**
When the user provides the voice command as 'Click <placeholder text>' the focus should go to the PnP search box. The PnP search box already has aria label and placeholder attributes and the voice over tool does not recognise the control using those values. As a fix, if there is a name attribute with the value same as whatever is being configured in the placeholder text by the user, then the tool will be able to recognise the control using the value in the name attribute.
If the user has set the placeholder text as Search news, then if the name attribute can get the same placeholder text as its value, the dragon tool will be able to recognise the voice command and the focus would go to the search box control.

**Screenshots**
![134520468-b02a7939-127d-4075-b282-1be4796f992d](https://user-images.githubusercontent.com/90712499/153660106-6378c935-9772-4e64-8faf-8dd3fd41cc1e.png)

**Desktop (please complete the following information):**

**Comment by MariaAug** [comment]
@wobba - Hi, is there any update on this issue fix please?

**Comment by wobba** [comment]
When someone has time to look at it and implement it :) Personally my spare time is quite limited these days.
